book/glossary.md

```diff
@@ -83,59 +101,86 @@ __HTML__.
 __Element__: Most __Nodes__ in the __DOM__ tree are Elements (except for
 text and the document object). (Inherits from __Node__.)
 
-__Focus__: the property of an __element__ being the highlight, or "focus",
+__Event__: A way for JavaScript to observe that something has happened on the
+document, and customize its results.
+
+__Focus__: Ahe property of an __element__ being the highlight, or "focus",
 of user interaction, and therefore receiving keyboard events and being
 visually highlighted on the screen.
 
-__IFrame__: a way of embedding a child __document__ within a parent, through
+__Font__: A particular stylistic way of drawing a particular human language to
+computer screens. Arial is one common example for Latin-based langauges.
+
+__IFrame__: A way of embedding a child __document__ within a parent, through
 a rectangular window on the screen reserved for it that participates in the
 layout of the parent.
 
+__Image__: A representation of a picture to draw on a computer screen. An
+HTML element of the same name, for the same purpose.
+
 __Node__: A point in the __DOM__ tree, with parent and child pointers.
 
 __Page__: The conceptual container for a __document__. Unlike its physical
 analogue, a page can have multiple documents (through use of __iframes__).
 
+__Style sheet__: A document resource that contains __CSS__ rules.
 
-Box model
-=========
+__Tag__: The name of a particular type of HTML element, indicating its
```

Comment:
Ok, this is super duper nit-picky, but this is defining the "tag name", not the "tag". The way the HTML standard uses these terms (see, say, https://html.spec.whatwg.org/multipage/syntax.html#elements-2) is that if you have

    <b>asdf</b>

Then this is one _element_ (and one text node), with two _tags_ (start and end), each of which has _tag name_ `b`.